src/node.js

```diff
@@ -215,8 +215,30 @@
   startup.globalVariables = function() {
     global.process = process;
     global.global = global;
-    global.GLOBAL = global;
-    global.root = global;
+
+    // Deprecate GLOBAL and root
+    ['GLOBAL', 'root'].forEach(function(name) {
+      // getter
+      const get = NativeModule.require('util').deprecate(function() {
+        return this;
+      }, "'" + name + "' is deprecated, use 'global'");
+      // setter
+      const set = NativeModule.require('util').deprecate(function(value) {
+        Object.defineProperty(this, name, {
+          configurable: true,
+          writable: true,
+          enumerable: true,
+          value: value
+        });
+      }, "'" + name + "' is deprecated, use 'global'");
+      // define property
+      Object.defineProperty(global, name, {
```

Comment:
You can write this shorthand as `Object.defineProperty(global, name, { get, set, configurable: true });` now.